index.js

```diff
@@ -80,6 +80,12 @@ Merry.prototype.router = function (opts, routes) {
           var statusCode = err.output.statusCode ||
             (res.statusCode >= 400 ? res.statusCode : 500)
 
+          if (statusCode >= 400) {
```

Comment:
nope, if it's `>= 500` it'll also be `>= 400` - the second condition will never resolve